src/app/components/register/register.component.ts

```diff
@@ -28,6 +28,7 @@ export class RegisterComponent implements OnInit {
   SubmitData() {
     this.model.name = this.profileForm.get("fullName").value;
     this.model.image = this.profileForm.get("photoURL").value;
+    console.log(this.model);
```

Comment:
No console logs in pull requests

Reply:
es provisional para comprobar que se envia el objeto

Reply:
o puedes ayudarme a poner un logger